src/uwtools/tests/test_schemas.py

```diff
@@ -327,11 +365,11 @@ def test_schema_cdeps_atm_in(cdeps_config):
         "skip_restart_read",
     ]
     for k in ks_boolean:
-        assert "is not of type 'boolean'" in nmlerr(k, None)
+        assert "is not of type 'boolean'\n" in nmlerr(k, None)
```

Comment:
Add a newline to these type-error assertions to avoid accidental partial-line matches.